src/kit/taosdemo/taosdemo.c

```diff
@@ -6427,8 +6427,7 @@ static int64_t generateStbRowData(
             "(%" PRId64 "", timestamp);
 
     for (int i = 0; i < stbInfo->columnCount; i++) {
-        tstrncpy(pstr + dataLen, ",", 2);
-        dataLen += 1;
+        dataLen += snprintf(pstr + dataLen, 2, ",");
```

Comment:
printf family function has longer execution time lead performance decrease.

Reply:
I will change it back